test/e2e/apimachinery/webhook.go

```diff
@@ -682,9 +687,6 @@ func cleanWebhookTest(client clientset.Interface, namespaceName string) {
 	_ = client.ExtensionsV1beta1().Deployments(namespaceName).Delete(deploymentName, nil)
 	_ = client.CoreV1().Secrets(namespaceName).Delete(secretName, nil)
 	_ = client.RbacV1beta1().RoleBindings("kube-system").Delete(roleBindingName, nil)
-	_ = client.CoreV1().ConfigMaps(skippedNamespaceName).Delete(disallowedConfigMapName, nil)
```

Comment:
You need to remove the delete webhook configurations from the cleanWebhookTest

Reply:
Done.